modules/admission-webhook-create.adoc

```diff
@@ -0,0 +1,328 @@
+// Module included in the following assemblies:
+//
+// * architecture/admission-controllers.adoc
+
+[id="admission-webhook-create_{context}"]
+= Creating an admission webhook
+
+.Procedure
```

Comment:
This procedure heading should go directly before your steps. If you have introductory text or notes for a procedure module (which is common), that should come before the procedure heading.

Reply:
Resolved.